packages/itwin/imodel-react-hooks/package.json

```diff
@@ -50,30 +54,31 @@
     "verbose": true
   },
   "devDependencies": {
-    "@bentley/bentleyjs-core": "^2.0.0",
-    "@bentley/geometry-core": "^2.0.0",
-    "@bentley/imodeljs-common": "^2.0.0",
-    "@bentley/imodeljs-frontend": "^2.0.0",
+    "@itwin/build-tools": "^3.0.0",
+    "@itwin/core-bentley": "^3.0.0",
+    "@itwin/core-geometry": "^3.0.0",
+    "@itwin/core-common": "^3.0.0",
+    "@itwin/core-frontend": "^3.0.0",
     "@testing-library/react": "^9.1.4",
     "@types/jest": "^24.9.0",
-    "@types/react": "~16.9.35",
-    "@types/react-dom": "^16.8.0",
+    "@types/react": "^17.0.19",
+    "@types/react-dom": "^17.0.9",
     "@types/testing-library__react": "^9.1.3",
     "babel-jest": "^24.9.0",
     "jest": "^26.1.0",
-    "react": "^16.13.0",
-    "react-dom": "^16.13.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-test-renderer": "^16.13.0",
     "rimraf": "^3.0.2",
     "ts-jest": "^26.1.1",
-    "typescript": "^3.9.2"
+    "typescript": "~4.4.0"
   },
   "peerDependencies": {
-    "@bentley/bentleyjs-core": "^2.0.0",
-    "@bentley/geometry-core": "^2.0.0",
-    "@bentley/imodeljs-common": "^2.0.0",
-    "@bentley/imodeljs-frontend": "^2.0.0",
-    "react": "^16.13.0",
-    "react-dom": "^16.13.0"
+    "@itwin/core-bentley": "^3.0.0",
+    "@itwin/core-geometry": "^3.0.0",
+    "@itwin/core-common": "^3.0.0",
+    "@itwin/core-frontend": "^3.0.0",
+    "react": "^17.0.2",
```

Comment:
won't this force consumers to need react 17? If this package wasn't changed to break support for react 16, why not: `^16.13.0||^17.0.2`?

Reply:
in 3.0 we decided to move all core and related pkgs to 17; no real reason anyone should stick on 16; for the most part its backwards compatible so users shouldnt have any difficulty updgrading

Reply:
I'm still kind of against forcing an upgrade although i did hear it's backwards compatible. But if core did that then go ahead, resolve as you wish

Reply:
we (core) didnt want to continue maintaining 2 majors of react; much easier for us to force users to move up with us